### DISCUSSION TITLE
Support GLTF2

### DISCUSSION THREAD

**Discussion by fire** [libigl/libigl#2392] [Ideas]
Hi, is there any interest in GLTF2 support in igl? I have written a gltf2 implementation elsewhere. So looking for interest.

**Comment by alecjacobson** [comment]
Hi, thanks for your interest. I guess I probably don't want to maintain a separate implementation inside of libigl, but depending on something stable like https://github.com/syoyo/tinygltf would be reasonable.

**Reply by fire** [reply]
I am not picky. Could use tinygltf.

The one I built for Godot Engine isn't expected to be extracted outside Godot Engine. https://github.com/godotengine/godot/blob/master/modules/gltf/gltf_document.h

**Reply by fire** [reply]
I was looking into the python ecosystem and it was difficult to find a gltf2 implementation that exposed enough details for me to write a UV and skinning weights importer exporter.

My project was to use the robust weight transfer library which uses igl and python.

https://github.com/rin-23/RobustSkinWeightsTransferCode
